pkg/edge/trace_observers_integration_test.go

```diff
@@ -0,0 +1,56 @@
+package edge
+
+import (
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	mock "github.com/newrelic/newrelic-client-go/internal/testing"
+)
+
+var (
+	testAccountIDStr      = os.Getenv("NEWRELIC_MASTER_ACCOUNT_ID")
```

Comment:
Should be `NEW_RELIC_MASTER_ACCOUNT_ID` to go along with the NR standard env vars

Reply:
👍